src/config.rs

```diff
@@ -385,28 +442,70 @@ mod tests {
         assert_eq!(format!("{}", internal_ips), "169.254.0.0/16,192.168.0.0/16,fc00:0:0:0:0:0:0:0/7,127.0.0.1/32,10.0.0.0/8,172.16.0.0/12");
     }
 
+    fn get_reader_sections(readers: Vec<ConfigReader>) -> Vec<String> {
+        let config = Config {
+            enable_stats: false,
+            enable_flows: false,
+            readers: readers,
+            ..Config::default()
+        };
+
+        let config_readers = config.config_readers();
+
+        let rendered = config.render(&config_readers).unwrap();
+
+        let mut result = vec![];
+
+        let mut outputs_section: String = rendered
+            .chars()
+            .skip(rendered.find("outputs:\n").unwrap())
+            .collect();
+
+        let eve_log = "  - eve-log:\n";
```

Comment:
I think instead of searching the eve-log, we should look at templating this information, possibly with multiple include files.

Reply:
This was pulling out sections for test assertions, do we need it to be that template-able?

Reply:
Same way we test output for http config:

https://github.com/protectwise/suricata-ipc/blob/master/src/config.rs#L388

Reply:
We would instead be checking individual materializations, rather than pulling out sections. Makes the config and template a little cleaner, and testing more straightforward. 

We'll tackle it in another PR.

Reply:
Ah, that makes sense

Reply:
ty for merge :smile: